chord_metadata_service/mcode/parse_fhir_mcode.py

```diff
@@ -0,0 +1,306 @@
+import uuid
+
+from .mappings.mappings import *
+from .mappings.mcode_profiles import *
+from chord_metadata_service.restapi.schemas import FHIR_BUNDLE_SCHEMA
+from chord_metadata_service.restapi.fhir_ingest import check_schema
+
+
+def get_ontology_value(resource, codeable_concept_property):
+    """
+    The function covers the most encountered use cases.
+     """
+    try:
+        ontology_value = {}
+        if "system" in resource[codeable_concept_property]['coding'][0]:
+            ontology_value["id"] = f"{resource[codeable_concept_property]['coding'][0]['system']}:" \
+                                   f"{resource[codeable_concept_property]['coding'][0]['code']}"
+        else:
+            ontology_value["id"] = f"{resource[codeable_concept_property]['coding'][0]['code']}"
+
+        if "display" in resource[codeable_concept_property]['coding'][0]:
+            ontology_value["label"] = f"{resource[codeable_concept_property]['coding'][0]['display']}"
+        else:
+            ontology_value["label"] = f"{resource[codeable_concept_property]['coding'][0]['code']}"
+        return ontology_value
+    # will be raised if there is no "code" in Coding element
+    except KeyError as e:
+        raise KeyError(e)
```

Comment:
what's the use of this, since it just raises the same error?

Reply:
my bad, I fixed it